lib/jsonapi/resource.rb

```diff
@@ -551,7 +551,7 @@ def has_one(*attrs)
       end
 
       def belongs_to(*attrs)
-        ActiveSupport::Deprecation.warn "In #{name} you exposed a `has_one` relationship "\
```

Comment:
Nit:  Should probably keep.

Reply:
Dangit I restored that a couple of times but through rebasing and refactoring it snuck back in. I was as baffled as you probably were that this line was deleted. The PR was full of bizarre deletions and additions and dead-ends.